sources/org.osbuild.curl

```diff
@@ -154,6 +154,7 @@ class CurlSource(sources.SourceService):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.subscriptions = None
+        self._parallel_downloads = _curl_has_parallel_downloads()
```

Comment:
May as well squash caching this into the previous commit.

Reply:
I reworked the commits now.